file/file.proto

```diff
@@ -18,16 +18,24 @@ syntax = "proto3";
 
 package gnoi.file;
 
+import "github.com/openconfig/gnoi/common.proto";
 import "github.com/openconfig/gnoi/types.proto";
 
 service File {
-  // Get read and streams the contents of a file from the target.
+  // Get reads and streams the contents of a file from the target.
   // The file is streamed by sequential messages, each containing up to
   // 64KB of data. A final message is sent prior to closing the stream
   // that contains the hash of the data sent. An error is returned
   // if the file does not exist or there was an error reading the file.
   rpc Get(GetRequest) returns (stream GetResponse) {}
 
+  // GetRemote transfers the contents of a file from the target to a specified
+  // remote location. The response contains the hash of the data transferred.
+  // An error is returned if the file does not exist, the file transfer fails,
+  // or if there was an error reading the file. This is a blocking call until
+  // the file transfer is complete.
+  rpc GetRemote(GetRemoteRequest) returns (GetRemoteResponse) {}
```

Comment:
Wouldnt "TransferToRemote" or something similar a better name? Sorry not trying to be obsessed with the name, but having "Get" in the name of the RPC is a bit confusing.

Reply:
I'm pretty open to a different name. Maybe TransferFromRemote would be more obvious here?

@aashaikh @robshakir Any thoughts on the name here as well? I know we had discussed this previously as well.

Reply:
I'm fine with `TransferFromRemote` or even `FetchRemote` here, but I don't have strong feelings. 

The "get" in `GetRemote` is essentially "get this file from a remote place" - maybe fetch is better than "get" here?

Reply:
I'm leaning towards TransferFromRemote, but please let me know if you feel otherwise.

Reply:
After thinking again about it, I agree with @aghaffarkhah on TransferToRemote :)

I have updated using this name now. I think this is much more obvious than GetRemote.